mini-project/BillManagementSystem/accountings/models.py

```diff
@@ -0,0 +1,40 @@
+from djmoney.models.fields import MoneyField
+from phonenumber_field.modelfields import PhoneNumberField
+
+from django.db import models
+from django.contrib.auth.models import User
+
+
+# Create your models here.
+class Payment(models.Model):
+    user_id = models.ForeignKey(User, on_delete=models.CASCADE)
+    amount = MoneyField(max_digits=19, decimal_places=2, default_currency='MYR')
+    note = models.TextField()
+    created_at = models.DateTimeField()
+    updated_at = models.DateTimeField()
+    
+class Debt(models.Model):
+    user_id = models.ForeignKey(User, on_delete=models.CASCADE)
+    amount = MoneyField(max_digits=19, decimal_places=2, default_currency='MYR')
+    created_at = models.DateTimeField()
+    updated_at = models.DateTimeField()
+    
+class Category(models.Model):
+    name = models.TextField()
+    
+class Expenses(models.Model):
+    category = models.ForeignKey(Category, on_delete=models.CASCADE)
+    user_id = models.ForeignKey(User, on_delete=models.CASCADE)
+    amount = MoneyField(max_digits=19, decimal_places=2, default_currency='MYR')
+    note = models.TextField()
+    created_at = models.DateTimeField()
+    updated_at = models.DateTimeField()
+
+class Student(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    birthday = models.DateTimeField()
+    email = models.EmailField()
```

Comment:
Duplicate of field in django user model

Reply:
@kwongtn how can I know the default field available in `User`? Any docs on that?

Reply:
Found it . Line 358.
```python
email = models.EmailField(_("email address"), blank=True)
```